docs/tiltfile_config.md

```diff
@@ -111,6 +111,54 @@ if 'd' in to_edit:
     docker_build('d', './d')
 ```
 
+### Grouping services in web UI
```

Comment:
@ellenkorbes, i know we chatted about resource grouping explanation living in the tiltfile config docs, but i'm wondering if that's the best place for them, since this page seems mostly about configuring different ways for your tiltfile to be run. now that you see resource grouping here, does it seem out of place?

Reply:
> now that you see resource grouping here, does it seem out of place?

No, this is good!